protocol/app/msgs/unsupported_msgs.go

```diff
@@ -18,6 +20,41 @@ var (
 		"/cosmos.gov.v1beta1.MsgSubmitProposal":         &govbeta.MsgSubmitProposal{},
 		"/cosmos.gov.v1beta1.MsgSubmitProposalResponse": nil,
 
+		// cosmwasm
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  nil,
+		"/cosmwasm.wasm.v1.AcceptedMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.AllowAllMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.ClearAdminProposal":                         nil,
+		"/cosmwasm.wasm.v1.CombinedLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxCallsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxFundsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MsgClearAdmin":                              &wasm.MsgClearAdmin{},
+		"/cosmwasm.wasm.v1.MsgClearAdminResponse":                      nil,
+		"/cosmwasm.wasm.v1.MsgIBCCloseChannel":                         &wasm.MsgIBCCloseChannel{},
+		"/cosmwasm.wasm.v1.MsgIBCSend":                                 &wasm.MsgIBCSend{},
+		"/cosmwasm.wasm.v1.MsgPinCodes":                                &wasm.MsgPinCodes{},
```

Comment:
For MsgPinCodes, MsgUnpinCodes and responses: uses by governance to pin a contract code. We should allow these

Reply:
Update: actually since these are only used by governance, believe they can stay in `unsupported_msgs` here, which only blocks messages in ante handler